datasets/austin/bikeshare_trips/pipeline.yaml

```diff
@@ -0,0 +1,133 @@
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+resources:
+
+  - type: bigquery_table
+    # Required Properties:
+    table_id: bikeshare_trips
+
+    # Description of the table
+    description: "austin bikeshare trips"
+
+dag:
+  airflow-version: 2
+  initialize:
+    dag_id: bikeshare_trips
+    default_args:
+      owner: "Google"
+
+      # When set to True, keeps a task from getting triggered if the previous schedule for the task hasn’t succeeded
+      depends_on_past: False
+      start_date: '2021-03-01'
+    max_active_runs: 1
+    schedule_interval: "@daily"  # run once a week at Sunday 12am
+    catchup: False
+    default_view: graph
+
+  tasks:
+
+    - operator: "KubernetesPodOperator"
+
+      # Task description
+      description: "Run CSV transform within kubernetes pod"
+
+      args:
+
+        task_id: "transform_csv"
+
+        # The name of the pod in which the task will run. This will be used (plus a random suffix) to generate a pod id
+        name: "bikeshare_trips"
+
+        # The namespace to run within Kubernetes. Always set its value to "default" because we follow the guideline that KubernetesPodOperator will only be used for very light workloads, i.e. use the Cloud Composer environment's resources without starving other pipelines.
+        namespace: "default"
+
+        image_pull_policy: "Always"
+
+        # Docker images will be built and pushed to GCR by default whenever the `scripts/generate_dag.py` is run. To skip building and pushing images, use the optional `--skip-builds` flag.
+        image: "{{ var.json.austin.container_registry.run_csv_transform_kub_bikeshare_trips }}"
+
+        # Set the environment variables you need initialized in the container. Use these as input variables for the script your container is expected to perform.
+        env_vars:
+          # SOURCE_FILE: "{{ var.json.shared.airflow_home }}/data/austin_bikeshare_trips/bikeshare_trips/data.csv"
+          # TARGET_FILE: "{{ var.json.shared.airflow_home }}/data/austin_bikeshare_trips/bikeshare_trips/data_output.csv"
+          SOURCE_URL: "https://data.austintexas.gov/api/views/tyfh-5r8s/rows.csv"
+          SOURCE_FILE: "/custom/data.csv"
+          TARGET_FILE: "/custom/data_output.csv"
+          TARGET_GCS_BUCKET: "{{ var.json.shared.composer_bucket }}"
+          TARGET_GCS_PATH: "data/austin/bikeshare_trips/data_output.csv"
+
+
+        # Set resource limits for the pod here. For resource units in Kubernetes, see https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#resource-units-in-kubernetes
+        resources:
+          limit_memory: "8G"
```

Comment:
I wonder if 4GB memory or less is sufficient?

Reply:
Done